### ISSUE TITLE
Steam fails to start on Kubuntu 16.10 with Kaby Lake CPU

### ISSUE THREAD

**Issue by christianhujer** [ValveSoftware/steam-for-linux#4838]
#### Your system information

* Steam client version: Downloaded latest version today. Unable to determine version because it fails to start. Also, tried `steam --version`, `steam -version`, `steam -v` - unsuccessfully. Here's the md5sum of `steam.deb`:
```
christian.hujer@christian-hujer-Blade-Stealth:~$ md5sum ~/Downloads/steam_latest.deb 
826ce064640e5ffc3c54bd8cfa3032e4  /home/christian.hujer/Downloads/steam_latest.deb
```
* Distribution (e.g. Ubuntu): Kubuntu 16.10 (Ubuntu 16.10 with KDE)
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No] Yes

Additional system information:
The machine is a Razer Blade Stealth with Intel Core i7-7500U (Kaby Lake), 16 GB RAM, 1 TB SSD, UHD 4K display (3840×2160).

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large code pastes as a [Github Gist](https://gist.github.com/)

In short: Steam fails to start.

Given the machine is a Razer Blade Stealth,
And Kubuntu 16.10 is installed,
And the system is uptodate,
And the latest steam client is installed,
When the steam client is run,
Then (expected behavior) the steam client should show up as a tray icon or in a window.
Then (actual behavior) there are error messages on the command line and no UI element shows up (no window, no tray icon).

Output:
```
christian.hujer@christian-hujer-Blade-Stealth:~$ steam
Running Steam on ubuntu 16.10 64-bit
STEAM_RUNTIME is enabled automatically
/bin/bash: /home/christian.hujer/.local/share/Steam/ubuntu12_32/steam-runtime/amd64/lib/x86_64-linux-gnu/libtinfo.so.5: no version information available (required by /bin/bash)
grep: symbol lookup error: grep: undefined symbol: pcre_jit_stack_alloc
grep: symbol lookup error: grep: undefined symbol: pcre_jit_stack_alloc
grep: symbol lookup error: grep: undefined symbol: pcre_jit_stack_alloc
Installing breakpad exception handler for appid(steam)/version(1482202200)
libGL error: unable to load driver: i965_dri.so
libGL error: driver pointer missing
libGL error: failed to load driver: i965
libGL error: unable to load driver: swrast_dri.so
libGL error: failed to load driver: swrast
```

#### Steps for reproducing this issue:
1. Install Kubuntu 16.10 on Razer Blade Stealth
2. Install steam.
3. Start steam.

**Comment by kisak-valve** [comment]
Hello @christianhujer, you are experiencing a steam runtime compatibility issue where one or more steam runtime libraries are too old to be used with the system-installed mesa. This is being tracked at #4768.

**Comment by christianhujer** [comment]
@kisak-valve Thank you for linking to #4768. The advice under section *Collection of release specific workarounds* helped. The advice was:

Ubuntu 16.10 with Mesa:
`LD_PRELOAD='/usr/lib/i386-linux-gnu/libstdc++.so.6 /lib/i386-linux-gnu/libgpg-error.so.0' steam`
There are tons of warning and error messages coming up, however, the steam client starts successfully and seems to be fully usable and fully functional.

I suggest that this issue is closed as a duplicate of #4768.
